ostart_contacts/essai_intent.eliom

```diff
@@ -0,0 +1,197 @@
+(* Service for this demo *)
+let%server service =
+  Eliom_service.create
+    ~path:(Eliom_service.Path ["essai-intent"])
+    ~meth:(Eliom_service.Get Eliom_parameter.unit)
+    ()
+
+(* Make service available on the client *)
+let%client service = ~%service
+
+(* Name for demo menu *)
+let%shared name () = [%i18n S.essai_intent_button]
+
+(* Class for the page containing this demo (for internal use) *)
+let%shared page_class = "os-page-essai-intent"
+
+
+let%client l = ref []
+let%shared uri = ref ""
+let%shared is_video = ref false
+let%shared is_image = ref false
+let%shared is_text = ref false
+
+
+(*share a picture with the project and the link will be added to the list l*)
+let%shared affect_to_uri () =
+
+
+  let _ : unit Eliom_client_value.t =
+    [%client
+
+      let g (intent : Cordova_intent.intent) =
+        match (Cordova_intent.action intent) with
+          "android.intent.action.SEND" ->
```

Comment:
To avoid other people to make unecessary commits, and to facilitate greping, please respect coding style :
```match ... with
| "lkjlkj" -> ...
```
(bar on first line and space after bar).